### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by DarrenAlex**
Added delegated facts to the documentation

##### SUMMARY
Added a reference to Delegated Facts (https://docs.ansible.com/ansible/latest/user_guide/playbooks_delegation.html#delegated-facts) due to one user having difficulty finding information that was in Delegated Facts. (Issue #67067 https://github.com/ansible/ansible/issues/67067)
##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by samccann** [comment]
Hi @DarrenHSKG and thanks for contributing to Ansible!  This particular fix is to the wrong file so I'm going to close this PR out. But I hope you do open another with the correct file.

The original issue has a small section in the comments where the bot attempts to identify the correct file for you :
image

(Note - sometimes it's not accurate, but in this case it is).

I also apologize as I marked the issue `easyfix` but was forgetting it's not as simple as it seemed for people not familiar with the Ansible repository and files.

Since the module `setup.py` file is python code with an embedded DOCUMENTATION block, the way to add a URL is different.  We document how to do that [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation).  

Basically, you wrap the URL with L()... In this example, something like:

See L(Delegating Facts,https://docs.ansible.com/ansible/latest/user_guide/playbooks_delegation.html#delegated-facts).
